scripts/delete.js

```diff
@@ -0,0 +1,30 @@
+import { getTasksFromStorage, saveTaskToStorage } from "./add.js";
+
+document.getElementsByClassName("deleteEditButton")[0].addEventListener("click", () => {deleteTaskById(document.getElementsByClassName("deleteEditButton")[0].id)});
+
+/**
+ * Reads 'to-do-tasks' from localStorage, removes the task with the given ID
+ * And then rewrites the updated list of Tasks back to local storage.
+ * Then, it reloads the page to refresh the task list
+ * @param {number} id ID of the task to be removed from local storage
+ */
+export function deleteTaskById(id){
+
+    const taskList = getTasksFromStorage();
+    
+    //const del = document.getElementById('id');
+
+    for(var i = 0; i < taskList.length; i++) {
+        /**if(taskList[i] == del) {
+            taskList.splice(i, 1);
+        }**/
+
+        if(taskList[i].id == id){
+            taskList.splice(i, 1);
+        }
+    }
+
```

Comment:
Some comments would be nice here, describing what line 27 and line 29 does

Reply:
I think more so for line 29 -- line 27 could be waived if we added that line 23 removes the corresponding task from the list (in my opinion)